compiler-plugin/src/main/kotlin/arrow/meta/quotes/TypeAlias.kt

```diff
@@ -27,7 +27,7 @@ fun Meta.typeAlias(
  */
 class TypeAlias(
   override val value: KtTypeAlias,
-  val `@annotations`: ScopedList<KtAnnotationEntry> = ScopedList(value.annotationEntries),
+  val `@annotationEntries`: ScopedList<KtAnnotationEntry> = ScopedList(value.annotationEntries),
```

Comment:
I'm not sure what happened here but we want to preserve the name annotations vs annotationEntries.